numpy/distutils/_shell_utils.py

```diff
@@ -3,7 +3,7 @@
 parameters provided in config files.
 """
 import os
-import shlex
+from shlex import split as shlex_split
```

Comment:
LGTM does not like this type of code
```
import shlex
from shlex import ...
```

Reply:
Might be tempted to ignore it, we use that heavily in our `__init__.py`

Reply:
ok, will back this out